helloworld-impl/src/main/java/sample/helloworld/impl/HelloServiceImpl.java

```diff
@@ -37,32 +33,22 @@
   }
 
   @Override public ServiceCall<NotUsed, String> hello(String id) {
-    System.out.println("hello");
     return request -> {
-      // Look up the hello world entity for the given ID.
       PersistentEntityRef<HelloCommand> ref = persistentEntityRegistry.refFor(HelloWorld.class, id);
-      // Ask the entity the Hello command.
-      greetingsTopic();
       return ref.ask(new Hello(id, Optional.empty()));
     };
   }
 
   @Override public ServiceCall<GreetingMessage, Done> useGreeting(String id) {
-    System.out.println("usegreeting");
     return request -> {
-      // Look up the hello world entity for the given ID.
       PersistentEntityRef<HelloCommand> ref = persistentEntityRegistry.refFor(HelloWorld.class, id);
-      // Tell the entity to use the greeting message specified.
       return ref.ask(new UseGreetingMessage(request.message));
     };
 
   }
 
   @Override
-  //#implement-topic
   public Topic<GreetingMessage> greetingsTopic() {
-    System.out.println("greetingsTopic");
-    //throw new UnsupportedOperationException("Missing implementation");
     return TopicProducer.singleStreamWithOffset(offset -> persistentEntityRegistry.eventStream(HelloEventTag.INSTANCE, offset)
         .map(this::convertEvent));
   }
```

Comment:
the publishing of messages happens here. This code connects the event stream produced from the PersistentEntity to a Topic Producer. This event stream is an Akka Persistence Query which, by default, polls the event journal database every 10 seconds (that time is configurable). That means that every event emitted by the persistent entity will be seen by every read side and/or published eventually.

Now, to make sure something is being produced in the broker, it is necessary that something is emitted in the Persistent Entity. In the Hello sample code from the archetypes, the Persistent Entity publishes an event every time the Greetings messages is changed ( https://github.com/himani1/lagom-kafka/blob/master/helloworld-impl/src/main/java/sample/helloworld/impl/HelloServiceImpl.java#L42-L46  and https://github.com/himani1/lagom-kafka/blob/master/helloworld-impl/src/main/java/sample/helloworld/impl/HelloWorld.java#L65). 

In order to test the Broker message production and consumptions it is necessayr to start the application (`mvn lagom:runAll` or `sbt runAll`) and then:
 `curl -H "Content-Type: application/json" -X POST -d '{"message":"Hi"}' http://localhost:9000/api/hello/Alice` (see https://github.com/himani1/lagom-kafka/blob/master/helloworld-api/src/main/java/sample/helloworld/api/HelloService.java#L35)